webpack-demo/src/__mock__/addremove.test.js

```diff
@@ -0,0 +1,67 @@
+import Todos from '../todolist.js';
+
+document.body.innerHTML = `
+<div class="container">
+<h4 class="title">My Today's To do List</h4>
+<form>
+    <input class="description" type="text" placeholder="Add to your List..." autofocus />
+    <button class="add-btn">
+        <i class="fa-solid fa-arrows-rotate fa-icon"></i>
+    </button>
+</form>
+<div class="todos">
+</div>
+<div class="clear">
+    <a class="clear-btn">Clear All Completed Tasks</a>
+</div>
+</div>
+`;
```

Comment:
- [ ] In addition to the above comment, after I found the way to run the tests, The tests were failing. Look at the screenshot below 👇 
![image](https://user-images.githubusercontent.com/90222110/180090458-ec8ccbb4-5bf5-4112-a7ab-cfcdd8eb2d70.png)
You can see errors that they are flagging so please fix it and make sure your tests are passing. 😺 👍